packages/color-scale-block/src/components/SquareWithColor.tsx

```diff
@@ -0,0 +1,161 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useDrag } from 'react-dnd';
+import { DragSourceMonitor } from 'react-dnd';
+import { FC, MouseEvent, MouseEventHandler, useState } from 'react';
+import { Tooltip } from '@frontify/fondue';
+import { ItemDragState, TooltipAlignment, TooltipPosition } from '@frontify/fondue';
+import { ColorProps, SquareWithColorProps } from '../types';
+import { DragHandle } from './DragHandle';
+import { CustomizationOptionsModal } from './CustomizationOptionsModal';
+
+const rgbaToHex = (rgba: string, forceRemoveAlpha = false) => {
```

Comment:
Try `toHex8String` from the shared module.

Reply:
This was fixed as well in your commit. 👍